ios/app/brave_core_main.mm

```diff
@@ -269,6 +269,8 @@ - (instancetype)initWithUserAgent:(NSString*)userAgent
 }
 
 - (void)dealloc {
+  _backgroundImagesService = nil;
```

Comment:
It doesnt have anything to do with this PR but just wondering what about 
deAmpPrefs or ipfsAPI or adblockService

Reply:
Hmm I'm not sure about those ones. But without this, Brave-Core crashes upon termination as it doesn't destruct stuff in reverse order. Perhaps I should add the other ones. They didn't exist before this PR.